sdks/python/apache_beam/io/filebasedsource_test.py

```diff
@@ -183,16 +183,16 @@ def setUp(self):
     filebasedsource.MAX_NUM_THREADS_FOR_SIZE_ESTIMATION = 2
 
   def test_read(self):
-    sources = [TestConcatSource.DummySource(range(start, start + 10)) for start
-               in [0, 10, 20]]
+    sources = [TestConcatSource.DummySource(list(range(start, start + 10)))
```

Comment:
Do these really need to be changed? From looking at DummySource it seems like the operations it does should work fine on ranges.